include/logging/log_output.h

```diff
@@ -237,7 +237,7 @@ static inline void log_output_hostname_set(const struct log_output *output,
  *
  * @param freq Frequency in Hz.
  */
-void log_output_timestamp_freq_set(uint32_t freq);
```

Comment:
this is frequency of the timestamp and not the timestamp. Uint32_t should be enough (>4GHz).

Reply:
Yes, you're right. Besides, even in case (quite remote) we want to address a situation with frequency higher than 4GHz, the type name "log_timestamp_t" wouldn't be correct.
Sorry for this: I was looking at timestamps and then I ended up doing extra useless changes